modules/ROOT/pages/setup.adoc

```diff
@@ -0,0 +1,114 @@
+= Anypoint Partner Manager Setup
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+
+You should be familiar with the following concepts before setting up Partner Manager:
+
+* EDI message formats and schemas.
+* Dataweave transformations.
+* Digital Certificates (such as X.509) for the AS2 transport protocol.
+* Your own network of endpoints for the listening to and sending of messages.
+* Troubleshooting in the Anypoint platform.
+
+== Minimum Requirements
+
+Verify that you have the following software and other requirements.
+
+* Mule runtime engine 4.1.1 or higher.
+
+**Note**
+
+Make sure you have the needed Mule runtime Core/Vcore capacity to handle the B2B transaction load. If you are unsure about the capacity you need to handle B2B transactions, please contact your MuleSoft account representative for assistance.
+
+* Storage for your B2B message payloads. 
+
+* Anypoint Studio 7.1 or higher for creating DataWeave maps.
+
+* License for the following optional Connectors:
+
+** AS2 4.0.2 or higher. 
+** X12 2.0.1 or higher. 
+
+* Partner Manager Requirements:
+** Anypoint Platform user account with Anypoint Partner Manager access.
+** Appropriate roles allocated for users to access Anypoint Partner Manager. Roles are allocated via the Access Management section in Anypoint Platform.
+
+**Note**
+
+If you are unable to access Anypoint Partner Manager, please contact your MuleSoft account representative for assistance.
+
+== Setup Checklist
+
+Before you set up your message flows to send and receive messages with your partners, you'll need to collection some information from them. You’ll also need to understand how your backend systems work. Use this checklist to help organize the information you need. 
```

Comment:
Understanding is not a task like collecting info is. I'd leave it at "you'll need to collect [not collection] some information" (how can I collect info about flows that don't exist yet?).